x/group/go.mod

```diff
@@ -0,0 +1,15 @@
+go 1.15
+
+module github.com/cosmos/cosmos-sdk/x/group
+
+require (
+	github.com/cosmos/cosmos-sdk v0.43.0
```

Comment:
you probably need to add `replace github.com/cosmos/cosmos-sdk => ../../` too, otherwise you'll get some build errors in `x/group/proposal.go` because it cannot find `tx.GetMsgs` et al.